product-service/src/functions/createProduct/createProduct.ts

```diff
@@ -0,0 +1,39 @@
+import { APIGatewayProxyEvent } from "aws-lambda";
+import { validateNewProductData } from "../../utils/data.validation.utils";
+import { HttpCode } from "../../utils/http.utils";
+import { formatJSONResponse } from "../../libs/api-gateway";
+import { getClient } from "../../utils/connection.utils";
+import { createQueryInsertIntoProducts, createQueryInsertIntoStocks } from "./query";
+
+export const handler = async (event: APIGatewayProxyEvent) => {
+  console.log(event);
```

Comment:
Just a suggestion: instead of adding console log for every function you can create wrapper function which will log that data for you